### DISCUSSION TITLE
作者是否愿意把 UmamusumeDeserializeDB5 做成可执行程序让玩家自己去更新事件？

### DISCUSSION THREAD

**Discussion by trojan-uma** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#33] [General]
按照作者的说法UmamusumeDeserializeDB5是用于生成UmamusumeResponseAnalyzer基础数据的工具，目前并未见编译成可直接执行的程序（指未发布releases），普通玩家无法“开袋即食”。

既然工具获取的数据也来自第三方网站，能否作者把DB5编译成可执行程序然后供玩家自己生成数据，这样玩家不需要等待作者更新.json。

**Comment by EtherealAO** [comment]
因为没必要，大部分情况下我都会在kamigame更新后（或者有人告诉我更新了之后）重新生成，现在也只支持了kamigame的数据源，为了提前那段时间单独进行更新发布意义不大
